tests/drivers/build_all/sensor/i2c.dtsi

```diff
@@ -843,3 +843,9 @@ test_i2c_bmi08x_gyro: bmi08x@6e {
 	gyro-hz = "1000_116";
 	gyro-fs = <1000>;
 };
+
+test_i2c_ist8310@6f {
+	compatible = "isentek,ist8310";
+	reg = <0x6f>;
+	status = "okay";
+};
```

Comment:
(I won't block on this one)

This is a good start to make sure that the driver builds at least, ideally we'd see an emulator with the driver (not yet a hard requirement in Zephyr). You can see https://github.com/zephyrproject-rtos/zephyr/tree/main/drivers/sensor/akm09918c as a sample of a similar device's emulator.

Reply:
Awesome thanks, we will look to add the emulation down the road with some other sensors as well.